addons/cargo/functions/fnc_loadItem.sqf

```diff
@@ -35,11 +35,37 @@ _vehicle setVariable [QGVAR(space), _space - _itemSize, true];
 
 if (_item isEqualType objNull) then {
     detach _item;
-    _item attachTo [_vehicle,[0,0,-100]];
-    [QEGVAR(common,hideObjectGlobal), [_item, true]] call CBA_fnc_serverEvent;
+    if !(_vehicle setVehicleCargo _item) then {
+        private _itemsCargo = _loaded arrayIntersect getVehicleCargo _vehicle;
```

Comment:
Ideally, this _itemsCargo array should be sorted by the objects' surface area. Something like: 
`private _itemsCargo = [_loaded arrayIntersect getVehicleCargo _vehicle, [], {_x call FUNC(getSurfaceItem)}, "DESCEND"] call BIS_fnc_sortBy;` (Untested)
That should make things more predictable and maximize the number of objects that stay in ViV cargo.

Reply:
I didn't took a deep look to your suggestion but as `getVehicleCargo` :
> The returned array is in the order the cargo vehicles were added to containing vehicle.

So first loaded items will be stored as cargo net as you can see in PR description under the concept section. This make sure to have all cargo nets properly stacked
If we are starting to store items by taking them not in this order, cargo net will start to appear randomly in the ViV. At the end, you may see cargo nets with space between them (which is not "nice" but acceptable?)

I will take a look to see how this behave

Reply:
I did some testing and I didn't realize the other VIV items stay in place when the box gets added. 
In that case it is probably best to remove them in the order they were loaded like you sad, I withdraw my suggestion.

It might be a good idea to look into making some kind of dynamic rearrangement but that should probably be done in another PR.